scraper-ngoindia/app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+from scraper import get_page_data
```

Comment:
You should only have to import get_page_data from scraper since that's the only one you call